src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -57,5 +83,142 @@ public void SendAsync_SimpleGet_Success()
                     "Expected cancellation requested on token attached to exception.");
             }
         }
+
+        [Fact]
+        public void GetAsyncAutomaticDecompression()
+        {
+            var handler = new HttpClientHandler();
+            if (!handler.SupportsAutomaticDecompression)
```

Comment:
If you look at our ToF tests for the Http component, you will see that we test (validate) in the HttpClientHandler Constructor test that the property always return TRUE.